### DISCUSSION TITLE
where do we contribute to VA OVA system configurations

### DISCUSSION THREAD

**Discussion by korenlev** [Seagate/cortx#473] [Ideas] [answered]
for things like : 
init scripts and services
automation (like: ovs import configurations to include hostname and network settings options etc)

i mean a repo from which the OVA will be built from ?

**Comment by johnbent** [comment, marked as answer]
Hey Koren,

Currently the OVA is being built from all the submodules.  This question is somewhat related to this Issue, https://github.com/Seagate/cortx/issues/204, which currently points into a private repo.  I'm looking into why that repo is private and whether and how we can share our build processes with the community.
